metricbeat/mb/module/wrapper.go

```diff
@@ -311,23 +317,39 @@ func writeEvent(done <-chan struct{}, out chan<- common.MapStr, event common.Map
 	}
 }
 
-func getMetricSetExpvarMap(module, name string) (*expvar.Map, error) {
-	key := fmt.Sprintf("%s-%s", module, name)
+func getMetricSetStats(module, name string) *stats {
+	key := fmt.Sprintf("metricbeat.%s.%s", module, name)
```

Comment:
That means that if a module is configured multiple times with the same metricset but different periods, the stats will be combined?